data/elan-2cf7.tablet

```diff
@@ -0,0 +1,13 @@
+[Device]
```

Comment:
Add a comment header like in the other files please, but to pass the CI we'll need the sysinfo link. See https://github.com/linuxwacom/wacom-hid-descriptors/ and a recent commit for guidance (e.g. 9d61a6c0cf4c0052a53afa288a4bf900e4eb2018)